### ISSUE TITLE
Project moved to new repository

### ISSUE THREAD

**Issue by Methuselah96** [zalmoxisus/redux-devtools-extension#831]
Note that this project has been moved to https://github.com/reduxjs/redux-devtools. Please file your issues and PRs there.

**Comment by tebeco** [comment]
@zalmoxisus @Methuselah96 should this repo be archived ?
That might redirect user faster to the new repo.
and the describtion updated:
![image](https://user-images.githubusercontent.com/2266487/150278586-1d523939-737f-487e-b72c-cb70e10ffe49.png)

> When a repository is archived, its issues, pull requests, code, labels, milestones, projects, wiki, releases, commits, tags, branches, reactions, code scanning alerts, comments and permissions become read-only. To make changes in an archived repository, you must unarchive the repository first

**Comment by Methuselah96** [comment]
Yeah, it should be, I just don't currently have admin access to archive it at the moment. I'll email Mihail and see if I can get admin access to archive it or maybe he'll see this here.
